src/main/java/org/prebid/server/bidder/criteo/model/CriteoRequest.java

```diff
@@ -0,0 +1,27 @@
+package org.prebid.server.bidder.criteo.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.prebid.server.proto.openrtb.ext.request.ExtUserEid;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder
+@Value
+public class CriteoRequest {
+
+    String id;
+
+    CriteoPublisher publisher;
+
+    CriteoUser user;
+
+    @JsonProperty("gdprconsent")
```

Comment:
Criteo expect fields to come without camel or snake case*

Reply:
Minor. In bidder specific models we can just use lowercase for fields name: `CriteoGdprConsent gdprconsent`
But up to you.